Added support for Beaufort wind speed

murderbeard opened the pull request:
Added Beaufort (bft) as an option for wind units. Bft is often used in the Netherlands for weather forecast wind speeds.

Necessary functions for conversion from km/h to bft, unit text, exports and settings stuff added.

Confirmed working in the hourly forecast view and bottom text. There are some km/h's that seem fixed though, like the local forecast, but that's a different problem.

<img width="980" height="791" alt="image" src="https://github.com/user-attachments/assets/3f8951bd-bead-42c6-8087-03a7bff6f0d3" />

mwood77 commented:
Local forecast is converted: 
https://github.com/mwood77/ws4kp-international/blob/e26f66ae6dd28ac73f62ff0e1d4ca75d6b3f9bcd/server/scripts/modules/utils/localForecastTextGenerator.mjs#L42

Did you see it somewhere else?

<hr/>

The code seems fine, but you’ll need to bump the version to `11.4.0` and then attach a new build for release. 

1. You can bump the version here:

https://github.com/mwood77/ws4kp-international/blob/e26f66ae6dd28ac73f62ff0e1d4ca75d6b3f9bcd/package.json#L3

2. To build the project, run `npm run build` and attach all of the changed files. CD will hand deployment when merged.

<hr/>

Once you bump the version and attach the build, it’ll be fine to merge.

murderbeard commented:
Thanks for reply! I've committed the version bump and changed files from the npm run build.

I must have misremembered because as I was checking now the local forecast does indeed show bft. So false alarm.

To be able to build on Windows I've added a separate build command for Windows only, but I'm not sure if you want that included here or as a separate PR maybe.

murderbeard commented:
@mwood77 Cool, but I might be missing something but I don't see a way to perform the merge. I can only write more comments or 'close with comment' the PR. I don't have any write privileges directly but I thought maybe after the changes were reviewed the option would become available or something.

<img width="1423" height="389" alt="image" src="https://github.com/user-attachments/assets/4ed16f65-6a3c-4871-b21a-e3749863f5dd" />
